src/Hyperwallet/Util/ApiClient.php

```diff
@@ -216,4 +218,15 @@ private function checkResponseHeaderContentType($response) {
     public function putMultipartData($partialUrl, array $uriParams, array $options) {
         return $this->doRequest('PUT', $partialUrl, $uriParams, $options);
     }
+
+    private function uuid()
```

Comment:
Can we move this to a utility PHP class?

Reply:
Created the utility class HyperwalletUUID and added HyperwalletUUIDTest for code coverage.